### PR TITLE
some tweaks

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

File: thoth/prispevek.tex [hunk 1]
```diff
@@ -40,38 +40,38 @@
...
+Over the past years, there has been a hype with regards to AI and machine learning. Machine learning and AI are used in production systems with a lot of applications. they require significant effort to ensure applications behave correctly and are fully operational. In this paper we describe a recommendation system for AI and machine learning applications which use popular open source machine learning libraries. The described system, Thoth, is capable of storing observations based on which it predicts possible misbehavior in application, application assembling errors or issues when integrating application with other components in a single deployment.
```
Comment (fridex) [inline thoth/prispevek.tex:43]: s/lot of applications. they require/lot of applications. **T**hey require/

File: thoth/prispevek.tex [hunk 2]
```diff
@@ -40,38 +40,38 @@
...
+To build complex applications, there is often a need to create small building blocks that encapsulate required logic into an easy to access interface. These small units are often reusable and thus create a separate libraries used by libraries themselves as well as by the overarching, complex application. The described dependency chaining creates direct and most often also transitive dependencies (also know as an application software stack) for an application that is handled by a dependency management system in the given language ecosystem. Dependency management system hepls to resolve dependencies given a specified version range to meet required functionality offered by required dependencies.
```
Comment (fridex) [inline thoth/prispevek.tex:53]: s/hepls/helps/ but that is my issue :)

Comment (fridex) [inline thoth/prispevek.tex:53]: > `overarching, complex`

not sure, but IMHO the comma is not necessary

File: thoth/prispevek.tex [hunk 3]
```diff
@@ -40,38 +40,38 @@
...
 
-\subsection{Modelling Application Stacks in a Graph Database} \label{section_modelling_app_stacks}
+\subsection{Modelling Software Stacks in a Graph Database} \label{section_modelling_software_stacks}
 
-Considering dependencies, their relations between each other, the whole application can be modeled in a graph database. We chosed JanusGraph graph database to be our data store for storing data in a graph form.
+Considering dependencies, their relations between each other, the whole application can be modeled as a directed acyclic graph. We chosed JanusGraph~\cite{ref_janusgraph} to be our data store for storing data in a graph form.
```
Comment (fridex) [inline thoth/prispevek.tex:61]: It could be even cyclic graph - we handle that.

Comment (goern) [reply thoth/prispevek.tex:61]: right...

File: thoth/prispevek.tex (hunk 1 above)
Comment (goern) [reply thoth/prispevek.tex:43]: hmm, seems to be right :)

**Pull request closed without merging**